packages/@aws-cdk-testing/cli-integ/resources/cdk-apps/app/app.js

```diff
@@ -272,6 +288,24 @@ class ImportableStack extends cdk.Stack {
         });
       }
     }
+
+    if (process.env.INCLUDE_NODEJS_FUNCTION_LAMBDA === '1') {
+      new node_lambda.NodejsFunction(
+        this,
+        'cdk-import-nodejs-lambda-test',
+        {
+          bundling: {
+            minify: true,
+            sourceMap: false,
+            sourcesContent: false,
+            target: 'ES2020',
+            forceDockerBundling: true,
```

Comment:
In `cli-integ`, we don't keep a runtime dependency for `esbuild`. Uses docker `esbuild` instead.